src/background/services/monetization.ts

```diff
@@ -74,10 +82,13 @@ export class MonetizationService {
         receiver,
         connectedWallet,
         requestId,
+        tab,
         tabId,
         frameId,
```

Comment:
Should we just pass the `sender` and get `tabId` `frameId`, `tab` in `PaymentSession`?

Reply:
we need to call getSender here anyway, calling it again in the PaymentSession seemed to much